docs(admin-api) Content types support multipart. (#1889)
Kong/docs.konghq.com: app/2.0.x/admin-api.md

}
```

- **application/x-www-form-urlencoded**

Simple enough for basic request bodies, you will probably use it most of the time.
Note that when sending nested values, Kong expects nested objects to be referenced
with dotted keys. Example:

```
config.limit=10&config.period=seconds
```

- **multipart/form-data**

Similar to url-encoded, you will use dotted keys to reference nested objects.

```
    -F "config.limit=10"
    -F "config.period=seconds" 

> ```

Reviewer: Since `-F` is a `curl` thing, should we provide a full `curl` example?

I think that we can also mention that `multipart/form-data` is used more generally to send requests with files and binary data.

We can even show an example of configuring the `pre-function` serverless plugin:

```
curl -i -X POST http://localhost:8001/services/plugin-testing/plugins \
     -F "name=pre-function" \
     -F "config.functions=@custom-auth.lua"
```

https://docs.konghq.com/hub/kong-inc/serverless-functions/#with-a-database

Reviewer: Congrats on your first commit @baokong !

Reviewer 2: Yes, or if you want you can show it as:
```
--------------------------bfdd9fe2d3e813bb
Content-Disposition: form-data; name="config.period"

seconds
--------------------------bfdd9fe2d3e813bb
Content-Disposition: form-data; name="config.limit"

10
--------------------------bfdd9fe2d3e813bb--
```

Author: Yes, I will push another commit. Sorry I took so long, did not have notifications turned on for PR review.

}
```

- **application/x-www-form-urlencoded**

Simple enough for basic request bodies, you will probably use it most of the time.
Note that when sending nested values, Kong expects nested objects to be referenced
with dotted keys. Example:

```
config.limit=10&config.period=seconds
```

- **multipart/form-data**

> Similar to url-encoded, you will use dotted keys to reference nested objects.

Reviewer 3: ```suggestion
Similar to url-encoded, this content type uses dotted keys to reference nested objects.
```

> Here is an example of sending a Lua file to the pre-function Kong plugin.

Reviewer 3: ```suggestion
Here is an example of sending a Lua file to the pre-function Kong plugin:
```
